__main__.py

```diff
@@ -1,12 +0,0 @@
-import platform
-import subprocess
-import sys
-from pathlib import Path
-
-if __name__ == "__main__":
-    binary = Path(__file__).parent / "fpm"
-    if platform.system() == "Windows":
-        binary = binary.with_suffix(".exe")
-
-    res = subprocess.run([binary] + sys.argv[1:])
-    sys.exit(res.returncode)
```

Comment:
Why is this not needed anymore? I would prefer to have the possibility to use `python -m fpm` to launch a process, which is especially useful with venvs.

Reply:
will bring this back!